api_tests/institutions/views/test_institution_auth.py

```diff
@@ -219,6 +220,7 @@ def test_existing_user_found_but_not_affiliated(self, app, institution, url_auth
         user.reload()
         assert user.fullname == 'Foo Bar'
         assert institution in user.get_affiliated_institutions()
+        assert f'source:institution|{institution._id}' not in user.system_tags
```

Comment:
This is an question for product folks, do we update existing user with institution tags if they sign in via institution? Currently implementation in this PR doesn't and thus this test is correct.